src/commctl/client_script.py

```diff
@@ -57,12 +57,50 @@ def main():
     subcmd_parser.add_argument(
         '-r', '--rounds', type=int, default=12, help='Number of rounds')
 
+    user_data_parser = subparser.add_parser(
+        'user-data',
+        help='Generates a user-data file for use with cloud-init',
+        epilog='Example: commctl user-data -p -c my_cluster cluster.userdata')
+    user_data_parser.add_argument(
+        'outfile',
+        help='Path and file name to output the user-data contents')
```

Comment:
I'd prefer writing to standard output by default, and maybe have a `-o / --output FILE` convenience option.